### PR TITLE
update docker-compose for exo-community 6.3

### PR DESCRIPTION
**Pull request by Noussairhamrit**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,17 +1,7 @@
-version: '2'
+version: '2.3'
 services:
-  web:
```
Comment (bourasom) [inline docker-compose.yml:3]: maybe check Minimum usage of nginx without ssl

File: docker-compose.yml [hunk 2]
```diff
@@ -43,12 +53,26 @@ services:
...
+      - network.host=_site_
+
+  mongo:
+    image: mongo:4.4
+    container_name: mongo
+    ports:
+      - "127.0.0.1:27017:27017"
+    volumes:
+      - mongo_data:/data/db:rw
+      - ./conf/mongo.conf:/etc/mongod.conf:ro
```
Comment (bourasom) [inline docker-compose.yml:69]: is it really needed?

Comment (bourasom) [inline docker-compose.yml:69]:
```suggestion    
```

Comment (Noussairhamrit) [reply docker-compose.yml:69]: yes yes

**Review by bourasom — DISMISSED** [review]
LGTM

File: docker-compose.yml [hunk 3]
```diff
@@ -21,16 +21,26 @@ services:
...
+      EXO_MONGO_HOST: mongo
+      EXO_MONGO_PORT: 27017
+      EXO_MONGO_DB_NAME: chat
+      EXO_DEBUG_ENABLED: "true"
+    expose:
+      - "8080"
+      - "8000"
+    ports:
+      - "8080:8080"
+      - "8000:8000"
```
Comment (hbenali) [inline docker-compose.yml:36]:
```suggestion
      EXO_MONGO_HOST: mongo
      EXO_MONGO_PORT: 27017
      EXO_MONGO_DB_NAME: chat
    ports:
      - "8080:8080"
```

File: docker-compose.yml [hunk 4]
```diff
@@ -43,12 +53,25 @@ services:
...
+
+  mongo:
+    image: mongo:4.4
+    container_name: mongo
+    ports:
+      - "127.0.0.1:27017:27017"
+    volumes:
+      - mongo_data:/data/db:rw
+    restart: unless-stopped
+
```
Comment (hbenali) [inline docker-compose.yml:70]:
```suggestion
```

**Review by hbenali — APPROVED** [review]

**Pull request merged**